Starting with version 3.8, grep no longer quietly ignores backslashes
preceding a white-space character.

--- a/src/Makefile
+++ b/src/Makefile
@@ -89,16 +89,16 @@
 WGET = wget --no-check-certificate --quiet -O
 PLUGINDIR = $(shell gimptool-2.0 --gimpplugindir)/plug-ins
 BASH_COMPLETIONSDIR = $(shell pkg-config --variable completionsdir bash-completion || echo $(PREFIX)/share/bash-completion/completions)
-VERSION = $(shell grep 'gmic_version\ ' gmic.h | tail -c4 | head -c3)
-VERSION1 = $(shell grep 'gmic_version\ ' gmic.h | tail -c4 | head -c1)
-VERSION2 = $(shell grep 'gmic_version\ ' gmic.h | tail -c3 | head -c1)
-VERSION3 = $(shell grep 'gmic_version\ ' gmic.h | tail -c2 | head -c1)
+VERSION = $(shell grep 'gmic_version ' gmic.h | tail -c4 | head -c3)
+VERSION1 = $(shell grep 'gmic_version ' gmic.h | tail -c4 | head -c1)
+VERSION2 = $(shell grep 'gmic_version ' gmic.h | tail -c3 | head -c1)
+VERSION3 = $(shell grep 'gmic_version ' gmic.h | tail -c2 | head -c1)
 SVERSION=$(VERSION1).$(VERSION2).$(VERSION3)
 SRC_PATH=$(shell pwd)
 
 # Check that versions of files 'CImg.h' and 'gmic.h' match.
 ifneq (,$(wildcard CImg.h))
-CIMG_VERSION = $(shell grep 'cimg_version\ ' CImg.h | tail -c4 | head -c3)
+CIMG_VERSION = $(shell grep 'cimg_version ' CImg.h | tail -c4 | head -c3)
 else
 CIMG_VERSION = $(VERSION)
 endif
